### ISSUE TITLE
Instability of weighted cross entropy

### ISSUE THREAD

**Issue by One-punch24** [JianghaoWu/SIFA-pytorch#4]
Dear Authors,

Thank you for this nice implementation using Pytorch.

There is one subtle issue with the definition of wegithed cross entropy loss. I believe `self.eps` should be utilized to avoid log(0). For example if all of the image is background class, then it will cause the problem of log(0).

Thank you!
